snapshots/wb/2023-08-14/pip_api.py

```diff
@@ -469,7 +469,7 @@ def calculate_percentile(p, df):
     df["distance_to_p"] = abs(df["headcount"] * 100 - p)
     df_closest = (
         df.sort_values("distance_to_p")
-        .groupby(["ppp_version", "country", "year", "reporting_level", "welfare_type"], as_index=False)
+        .groupby(["ppp_version", "country", "year", "reporting_level", "welfare_type"], as_index=False, sort=False)
```

Comment:
@Marigold This change was recommended by Copilot, to add `sort=False` for it to be faster